src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,491 @@
+package strawman
+package collection.mutable
+
+import scala._
+import scala.collection.{generic, mutable}
```

Comment:
Well, usually it fails because of this: https://travis-ci.org/scala/collection-strawman/builds/262556290?utm_source=github_status&utm_medium=notification

I have fixed the test